candid/root.did

```diff
@@ -44,6 +44,7 @@ type WithIdArg = record { id : nat64; witness : bool };
 type WithWitnessArg = record { witness : bool };
 type Witness = record { certificate : vec nat8; tree : vec nat8 };
 service : {
+  contract_id : () -> (principal) query;
```

Comment:
As previous naming convention uses `get_`

```suggestion
  get_contract_id : () -> (principal) query;
```

Reply:
I'm reserving get_contract_id for we do a proper implementation that has witness like all other method.